tests/test_routes.py

```diff
@@ -0,0 +1,74 @@
+
+def test_len_of_empty_list(client):
```

Comment:
It looks like this test is checking that if the database is empty, we return an empty list for the get all endpoint. How could we rename this test to include those details about the route and input?